widgets/views/galleryListEntryAdd.php

```diff
@@ -20,7 +20,7 @@
     <div class="panel panel-default">
         <div class="panel-body">
             <a class="backgroundSuccess" <?= yii\bootstrap\Html::renderTagAttributes($htmlOptions) ?> href="<?= $addActionUrl ?>">
-                <img class="padding15perc" src="<?= Yii::$app->getModule('gallery')->getAssetsUrl() . '/plus.svg' ?>" />
+                <img class="padding15perc" src="<?= Yii::$app->getModule('gallery')->getAssetsUrl() . '/plus.svg' ?>" alt="add gallery image" />
```

Comment:
Can you please use `Yii::t here?`

Reply:
Good day, @luke-!

Oh, sure. I have missed this one. Sorry about that.
I added this (and some more), please take a look at the last commits.

Thank you!